### ISSUE TITLE
Can't  install dictionary

### ISSUE THREAD

**Issue by marko-pi** [Predelnik/DSpellCheck#94]
I am using DSpellCheck for Notepad++ x64.  Notepad++ starts in administrator mode.

When I want to download dictionaries, I get error "Can't create directory C:" or "Can't create directory .", depending if I choose "Install for all users" or not.

It is working fine since I *manually* created directory %NOTEPAD++_DIR%\plugins\config\Hunspell.

**Comment by Predelnik** [comment]
Thanks! I fixed that and will close when new version will be released.

**Comment by marko-pi** [comment]
Thanks for the plugin and fast reaction.

**Comment by abdekker** [comment]
This issue appears to be back on a fresh installation of v8.6 x64 on Windows 10. Kept getting the following error when I tried to download the "English (United Kingdom)" dictionary:
    Can't create folder .plugins\config\Hunspell.

Manually creating the "Hunspell" folder in C:\Program Files\Notepad++\plugins\config made the error disappear, only to show a different error about dictionaries not being downloaded because "they could not be saved.

Resolved by closing NPP and re-launching in Administrator mode.
